### PR TITLE
CLB: Background type cards (30 May)

### LINKED ISSUE #579: Scion of Halaster

### LINKED ISSUE #580: Acolyte of Bahamut

### LINKED ISSUE #581: Agent of the Iron Throne

### LINKED ISSUE #583: Candlekeep Sage

### LINKED ISSUE #596: Dungeon Delver

### LINKED ISSUE #597: Far Traveler

### PR DESCRIPTION
**Pull request by Northmoc**
- closes #579
- closes #580 
- closes #581 
- closes #583 
- closes #596 
- closes #597

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/scion_of_halaster.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Name:Scion of Halaster
+ManaCost:1 B
+Types:Legendary Enchantment Background
+S:Mode$ Continuous | Affected$ Creature.IsCommander+YouOwn | AddReplacementEffects$ Draw | Description$ Commander creatures you own have "The first time you would draw a card each turn, instead look at the top two cards of your library. Put one of them into your graveyard and the other back on top of your library. Then draw a card."
+SVar:Draw:Event$ Draw | ReplaceWith$ DBDig | CheckSVar$ X | SVarCompare$ EQ0 | CheckSecondSVar$ Y | SecondSVarCompare$ EQ0 | Description$ The first time you would draw a card each turn, instead look at the top two cards of your library. Put one of them into your graveyard and the other back on top of your library. Then draw a card.
+SVar:DBDig:DB$ Dig | DigNum$ 2 | AnyNumber$ | DestinationZone$ Graveyard | LibraryPosition2$ 0 | SubAbility$ DBStoreNum
+SVar:TrigReset:DB$ StoreSVar | SVar$ Y | Type$ Number | Expression$ 1
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/scion_of_halaster.txt:7]: I'm not sure that's good enough

What if the commander changes controller mid turn?
The Y will prevent draw replacement for the other player even if he didn't draw any yet?

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/scion_of_halaster.txt:7]: Good call – I think I have a better solution

File: forge-gui/res/cardsfolder/upcoming/hardy_outlander.txt [hunk 2]
```diff
@@ -2,10 +2,8 @@ Name:Hardy Outlander
 ManaCost:1 G
 Types:Legendary Enchantment Background
 S:Mode$ Continuous | Affected$ Creature.IsCommander+YouOwn | AddTrigger$ AttacksPlayer | Description$ Commander creatures you own have "Whenever this creature attacks a player, if no opponent has more life than that player, another target creature you control gets +X/+X until end of turn, where X is this creature's power."
-SVar:AttacksPlayer:Mode$ Attacks | ValidCard$ Card.Self | Attacked$ Player | Execute$ TrigPump | CheckSVar$ Y | SVarCompare$ GEZ | TriggerDescription$ Whenever this creature attacks a player, if no opponent has more life than that player, another target creature you control gets +X/+X until end of turn, where X is this creature's power.
+SVar:AttacksPlayer:Mode$ Attacks | ValidCard$ Card.Self | Attacked$ Player | Execute$ TrigPump | Condition$ AttackedPlayerWithMostLife | TriggerDescription$ Whenever this creature attacks a player, if no opponent has more life than that player, another target creature you control gets +X/+X until end of turn, where X is this creature's power.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/hardy_outlander.txt:5]: Not sure this cleanup works

the other card that uses it cares about all player, but this one should ignore Allies?

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/hardy_outlander.txt]: Good catch – I moved this to #584

**Pull request merged**